tests/unittests/onnxImporterTest.cpp

```diff
@@ -206,3 +206,54 @@ TEST(onnx, importBatchBoxCox) {
     }
   }
 }
+
+/// Test loading ReplaceNaN op from an ONNX model.
+/// Test with arg value = 1.0.
+TEST(onnx, importReplaceNaN) {
```

Comment:
we probably should understand what's happening with ASAN test. But for loader test it's sufficient to check graph structure after loading. Actual operator test is run in the operatorTest.